src/nvidia.cc

```diff
@@ -569,20 +586,41 @@ int set_nvidia_type(struct text_object *obj, const char *arg)
 			break;
 			
 		default:						// Unknown/invalid argument
+			// Error printed by core.cc
 			return 1;
 	}
 	return 0;
 }
 
+// Check we are able to query tid and return the amount of targets present
+static inline int get_nvidia_target_count(Display *dpy, TARGET_ID tid){
+	int num_tgts;
+	if (!XNVCTRLQueryTargetCount(dpy, translate_nvidia_target[tid], &num_tgts)) {
+		//NORM_ERR("%s: Failed to query number of nvidia targets (dpy: %d, tid: %d)", __func__, dpy, tid);
```

Comment:
> Why's this commented out?

Originally the `CRIT_ERR`s in `get_nvidia_value` and `get_nvidia_string` were `NORM_ERR`s, so having two errors would spam the console a lot.

My mindset was to leave this commented, so that if anyone was struggling to debug something they had the option of uncommenting an extra error to see whether XNVCTRLQueryTargetCount had failed or simply set `num_tgts` to something < 1.

--------

I'll amend the commit to save the number of targets and print it in the `CRIT_ERR`s instead of having a secret error. Something like:

```C++
int target_count = get_nvidia_target_count(dpy, tgt);

if (target_count <1) CRIT_ERR("blah, blah, blah, (blah, target_count: %d)", target_count);
```